src/sobol_sensitivity.jl

```diff
@@ -60,6 +74,33 @@ function first_order_var(f,p_range,N,y0,v,p1,p2,p3)
     ys
 end
 
+function first_order_quad(f,p_range,y0,v,p1,quadalg)
+    ys = Array{typeof(y0.u)}(undef,length(p_range))
+    for i in 1:length(p_range)
+        y = zero(y0)
+        indices = [k for k in 1:length(p_range) if k != i]
+         
+        function f_q(x,p)
+            p1[indices] = x
+            f(p1,p)
+        end
+        prob1 = QuadratureProblem(f_q,[p_range[i][1] for i in indices],[p_range[i][2] for i in indices])
+        
+        function f__(x,p)
+            p1[i] = x
```

Comment:
indices?

Reply:
https://github.com/JuliaDiffEq/DiffEqSensitivity.jl/pull/105/files/9ee50445fe6f1dd97d0e771b935531f24013bf5f#diff-a5b8480a843d991444fd63346ec73c55R81

Reply:
yes, this line should use that

Reply:
No, if you look at the last equation in page 281 http://www.andreasaltelli.eu/file/repository/Making_best_use.pdf, this is the outer integral, so this will just be the `i`